### DISCUSSION TITLE
Decky Installer won't install on my Steamdeck

### DISCUSSION THREAD

**Discussion by mkisamefloweroflotus** [SteamDeckHomebrew/decky-loader#517] [Q&A]
installing Decky I get this message "rm: cannot remove ' / tmp / user_install_script.sh': No such file or directory"

even i sue console to install and don't install i have a lot issues 
Curl 6 could not resolve host github.com

**Comment by PartyWumpus** [comment]
'Could not resolve host github.com' either means your steam deck isn't connected to the internet, or there's something weird with your network. Try using a mobile hotspot

**Reply by mkisamefloweroflotus** [reply]
after update and check it is working thanks so much
